OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -18,7 +18,7 @@ namespace OpenRA.Mods.Common.Traits
 	public class RallyPointInfo : ITraitInfo
 	{
 		public readonly CVec RallyPoint = new CVec(1, 3);
-		public readonly string IndicatorPalettePrefix = "player";
+		[PaletteReference] public readonly string IndicatorPalettePrefix = "player";
```

Comment:
This references (I think) a player palette, so regular palette names are not valid here.